sdk/eventgrid/azure-eventgrid/tests/eventgrid_preparer.py

```diff
@@ -1,10 +1,8 @@
 import functools
-from devtools_testutils import PowerShellPreparer
+from devtools_testutils import EnvironmentVariableLoader
 
 from azure.mgmt.eventgrid.models import Topic, InputSchema, JsonInputSchemaMapping, JsonField, JsonFieldWithDefault
 
-from devtools_testutils.resource_testcase import RESOURCE_GROUP_PARAM
-
 EVENTGRID_TOPIC_PARAM = 'eventgrid_topic'
 EVENTGRID_TOPIC_LOCATION = 'westus'
```

Comment:
location is still westus so curious why you're changing them to east us?

Reply:
the resources I ran tests with were eastus